src/js/components/Anchor/StyledAnchor.js

```diff
@@ -13,9 +13,15 @@ const sizeStyle = props => {
   if (props.size) {
     const size = props.size || 'medium';
     const data = props.theme.text[size];
+    if (data) {
+      return css`
+        font-size: ${data.size};
+        line-height: ${data.height};
+      `;
+    }
     return css`
-      font-size: ${data.size};
-      line-height: ${data.height};
+      font-size: ${size};
+      line-height: normal;
```

Comment:
How is this change backward compatible?

Reply:
Orginally it was setting the `font-size` and `line-height` to the `data.size`  and the `data.height` but what I added was if `data` is not defined then it would set 
```
   font-size: ${size};
      line-height: normal;
```

`size` was never being set to anything before.. I was referencing the `text` size.
https://github.com/grommet/grommet/blob/master/src/js/components/Text/StyledText.js#L6

Reply:
mabe instead of setting it to `data.size` we can check if data exists then set that or else set it to just `size`